### ISSUE TITLE
Allow Custom Court Colors

### ISSUE THREAD

**Issue by annimukherjee** [kabaddiPy/kabaddiPy#4]
Introduce a parameter to allow users to customize the colors used in the court visualizations.

**Comment by LessonsWithSarfaraz** [comment]
Name: [MD SARFARAZ](https://github.com/LearnWithNewton/)
Contributor Role: GSSOC24 Contributor

Please assign this issue #4 to me.

Also, I'd like to know the following points:
[x] How to get started?
[x] resources to better understand the requirements/issue/feature to be implemented

**Comment by Claytonn7** [comment]
Hello @LearnWithNewton,

Please go through the documentation to familiarize yourself (function index is [here](https://kabaddipy.github.io/kabaddiPy/function-index/)) 
The visualization functions in code are from [Line 1117](https://github.com/kabaddiPy/kabaddiPy/blob/main/pypi/kabaddiPy/kabaddiPy.py#L1117) in kabaddiPy.py

Feel free to reach out in case of doubts!

**Comment by shirsakm** [comment]
As someone has not been assigned, can I work on this?

**Comment by annimukherjee** [comment]
@shirsakm Please work on it. It's a few lines of matplotlib code! Easy PR.

**Comment by shirsakm** [comment]
Yes, I am currently working on it. But, I am currently a little stuck on the correct way to add this parameter. Should it be added as parameter to the function `_plot_player_zones_grid(...)`? Is there any other way I should consider, and what would be the advantages of that?

I believe the former method is viable, as it does not cause a breakage in existing functionality.
